### PR TITLE
Laser Tag Gear

### PR DESCRIPTION
**Pull request by Geevies**
* Added helmets to the laser tag gear loadout.
* You can now alt-click laser tag armour to be able to set how many hits it will take before flooring you.
* You can now use a multitool on laser tag gear to change which colour it is and works against.
* Lowered the laser tag gun recharge rate to 4 seconds per shot, down from 8 seconds per shot.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/clothing/sets/laser_tag.dm [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	icon_state = "vest"
+	item_state = "vest"
+	contained_sprite = TRUE
+	blood_overlay_type = "armor"
+	body_parts_covered = UPPER_TORSO
+	slowdown = 0
+	armor = list(melee = ARMOR_MELEE_KNIVES,
+				laser = ARMOR_LASER_MINOR,
+				energy = ARMOR_ENERGY_MINOR
+			) // still somewhat sturdy, but ultimately not very
```
Comment (NonQueueingMatt) [inline code/modules/clothing/sets/laser_tag.dm:18]: this should not be blocking knives, also, newline after the list declaration

File: code/modules/clothing/sets/laser_tag.dm [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+	laser_tag_color = "blue"
+
+/obj/item/clothing/head/helmet/riot/laser_tag
+	name = "laser tag helmet"
+	desc = "A helmet in the form of a riot helm, made for high-impact laser tag gameplay."
+	icon = 'icons/clothing/kit/laser_tag.dmi'
+	icon_state = "helmet"
+	item_state = "helmet"
+	contained_sprite = TRUE
+	armor = list(melee = ARMOR_MELEE_KNIVES,
```
Comment (NonQueueingMatt) [inline code/modules/clothing/sets/laser_tag.dm:74]: ditto

**Review by NonQueueingMatt — APPROVED** [review]

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
that was not meant to be an approval

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**